scripts/geo_metadata.py

```diff
@@ -566,9 +568,15 @@ def main(dataset):
 	ordered_cols = [c for c in geo_samples.columns if not c.startswith(('read_','index_'))] + [c for c in geo_samples.columns if c.startswith(('read_','index_'))]
 	geo_samples = geo_samples[ordered_cols]
 
+	# Checking for presence / creating output folder and associated subfolder
+	if os.path.exists(output_dir) == False:
```

Comment:
Looks like os wasn't imported to this script, so we'll need to add that to the libraries imported.